server/apps/distributed-app/docs/modules/ROOT/pages/configure/redis.adoc

```diff
@@ -15,8 +15,8 @@ to get some examples and hints.
 | redisURL
 | the Redis URI pointing to Redis server. Compulsory.
 
-| cluster.enabled
-| Whether we should attempt connections to Redis as a cluster.Defaults to false.
+| redis.topology
+| Redis server topology. Defaults to standalone. Possible values: standalone, cluster, master-replica
```

Comment:
This is a breaking change.

IMO we should either provide retro-compatibility for `cluster.enabled`, or docs the breaking changes in `upgrade-instructions.md`.

Reply:
updated upgrade-instructions.md